packages/react-data-grid/src/ReactDataGrid.js

```diff
@@ -830,6 +831,7 @@ const ReactDataGrid = createReactClass({
         } else {
           this.setState({selected}, () => { this.scrollToColumn(idx); });
         }
+        this.props.onBeforeEdit();
```

Comment:
This will throw an error if not passed in as a props. Either add it as a default prop or check that it exists first